sycl/include/sycl/handler.hpp

```diff
@@ -448,8 +448,7 @@ class __SYCL_EXPORT handler {
           bool CallerNeedsEvent);
 #endif
   __SYCL_DLL_LOCAL handler(std::shared_ptr<detail::queue_impl> Queue,
-                           detail::queue_impl *PrimaryQueue,
```

Comment:
Note to reviewers: `PrimaryQueue` is always same as `Queue` since we are not using `SecondaryQueue` as fallback. So, I've also removed `PrimaryQueue` here.